app/controllers/jobs_controller.rb

```diff
@@ -0,0 +1,43 @@
+class JobsController < ApplicationController
+  before_action :load_job
+
+  def show
+    if @job.completed?
+      show_completed_job
+    elsif @job.errored?
+      show_errored_job
+    else
+      show_submitted_job
+    end
+  end
+
+  protected
+
+  def publicly_accessible?
+    true
+  end
+
+  private
+
+  def load_job
+    @job ||= TrackableJob::Job.find(params[:id])
+  end
+
+  def show_completed_job
+    if @job.redirect_to.present?
```

Comment:
Use a guard clause instead of wrapping the code inside a conditional expression.<br>Favor modifier `if` usage when having a single-line body. Another good alternative is the usage of control flow `&&`/`||`.